### PR TITLE
Add all supported keys to network-config.json

### PR DESCRIPTION
**Pull request by sudoryan**
Adds all supported keys to network-config

Keys taken from https://github.com/unstoppabledomains/unstoppable-domains-website/blob/master/backend/lib/blockchain/KeyToResolverKey.ts

Regexes taken from: https://github.com/unstoppabledomains/unstoppable-domains-website/blob/master/backend/lib/routes/record/crypto/post.ts#L32

Keys without a regex don't have a validationRegex

Task:
https://www.pivotaltracker.com/story/show/176703491

### CODE REVIEW

**Comment by aquiladev** [comment]
Smart contracts are agnostic from keys' set, I think we should not keep it on the level. I think it is related to resolution libraries.

**Comment by DeRain** [comment]
> Smart contracts are agnostic from keys' set, I think we should not keep it on the level. I think it is related to resolution libraries.

In this case would be better to move this to the separate artefact and each library will be able to pull it on update.

File: src/network-config/network-config.json [hunk 1]
```diff
@@ -1,98 +1,634 @@
...
+    },
+    "social.payid.name": {
+      "deprecatedKeyName": "payid",
+      "deprecated": false,
+      "validationRegex": "^[0-9a-zA-Z]+\\$[0-9a-zA-Z]+\\.[0-9a-zA-Z]+$"
+    },
+    "whois.email.value": {
+      "deprecatedKeyName": "email",
+      "deprecated": false,
+      "validationRegex": "/^(([^<>()[]\\.,;:s@\"]+(.[^<>()[]\\.,;:s@\"]+)*)|(\".+\"))@(([[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}])|(([a-zA-Z-0-9]+.)+[a-zA-Z]{2,}))$/"
```
Comment (aquiladev) [inline src/network-config/network-config.json:492]: @bogdan suggested to simplify email validator

File: src/network-config/network-config.json [hunk 2]
```diff
@@ -1,98 +1,634 @@
...
+          "address": "0x84214215904cDEbA9044ECf95F3eBF009185AAf4",
+          "legacyAddresses": []
         }
+      }
     }
-}
\ No newline at end of file
+  },
+  "keys": {
+    "crypto.BTC.address": {"deprecatedKeyName": "BTC", "deprecated": false},
```
Comment (aquiladev) [inline src/network-config/network-config.json:97]: deprecated flag can be skipped when "false"

File: src/network-config/network-config.json (hunk 1 above)
Comment (bogdan) [reply src/network-config/network-config.json:492]: My suggested regex:

```suggestion
      "validationRegex": "/\A[^@\s]+@[^@\s]+\z/"
```

**Review by bogdan — APPROVED** [review]

**Pull request merged**